doc/cephadm/troubleshooting.rst

```diff
@@ -161,3 +161,15 @@ This means that you must run a command of this form::
   ceph config set mon public_network <mon_network>
 
 For more detail on operations of this kind, see :ref:`deploy_additional_monitors`
+
+Accessing the admin socket
+--------------------------
+
+Each Ceph daemon provides an admin socket bypassing the
+MONs (See :ref:`rados-monitoring-using-admin-socket`).
```

Comment:
Does this mean "Each Ceph daemon provides an admin socket **that** bypasses the Mons" or "Each Ceph daemon provides an admin socket **by** bypassing the MONs"?

Reply:
the former

Reply:
ADded suggestion for ease :)